# CHANGES MADE for ungoogled-chromium: The import of
# //buildtools/deps_revision.gni
# in file build/config/posix/BUILD.gn
# was reverted since it
# - breaks linux_portable.
# - has no clear side-effects (the include only defines constants).
# - has nothing to do with fontconfig.
description: disable build commands for embedded fontconfig
author: Michael Gilbert <mgilbert@debian.org>

--- a/base/test/BUILD.gn
+++ b/base/test/BUILD.gn
@@ -204,7 +204,7 @@ static_library("test_support") {
     deps += [ "//third_party/fuchsia-sdk:zx" ]
   }
 
-  if (is_linux) {
+  if (false) {
     public_deps += [ ":fontconfig_util_linux" ]
     data_deps = [
       "//third_party/test_fonts",
@@ -360,7 +360,7 @@ if (is_linux) {
     ]
   }
 
-  if (current_toolchain == host_toolchain) {
+  if (false) {
     executable("generate_fontconfig_caches") {
       testonly = true
       sources = [
--- a/content/shell/test_runner/BUILD.gn
+++ b/content/shell/test_runner/BUILD.gn
@@ -176,7 +176,7 @@ group("resources") {
   if (is_mac) {
     deps += [ ":test_runner_bundle_data" ]
     data_deps += [ ":test_runner_bundle_data" ]
-  } else {
+  } else if (false) {
     deps += [ "//third_party/test_fonts" ]
     data_deps += [ "//third_party/test_fonts" ]
   }
